src/badguy/goldbomb.cpp

```diff
@@ -29,14 +31,29 @@
 #include "supertux/sector.hpp"
 #include "util/reader_mapping.hpp"
 
+const float HOP_HEIGHT = -250.f;
+const float REALIZE_TIME = 0.5f;
+
+// SAFE_DIST >= REALIZE_DIST
+const float REALIZE_DIST = 32.f * 8.f;
+const float SAFE_DIST = 32.f * 10.f;
+
+const float NORMAL_WALK_SPEED = 80.0f;
+const float FLEEING_WALK_SPEED = 180.0f;
+const int NORMAL_MAX_DROP_HEIGHT = 16;
+const int FLEEING_MAX_DROP_HEIGHT = 600;
```

Comment:
Those constants can be marked as `static` since they are only used in this file.

Reply:
I've seen them wrapped around an anonymous namespace, should I do that aswell?

Reply:
No need to